Python/bin/packUtil.py

```diff
@@ -1060,8 +1061,10 @@ def packTupleID(triplet):
          tmp = m.group(1)
          #print ("m.groups is ", m.groups())
          if len(tmp) > 7:
-            raise RuntimeError("Can't pack " + trkSub + " because it it is too long")
-         packedTrkSub = m.group(1)
+            print('Warning: trkSub ' + trkSub + ' is too long for obs80, removing the first character')
+            packedTrkSub = tmp[1:]
+         else:
+            packedTrkSub = m.group(1)
```

Comment:
Just a note about style/readability, I'd prefer to write this code block like:
```
if m:
   packedTrkSub = m.group(1)
   if len(packedTrkSub) > 7:
      print('Warning: trkSub ' + trkSub + ' is too long for obs80, removing the first character')
      packedTrkSub = packedTrkSub[1:]
```
to avoid the `tmp` variable. But what works, works.

Reply:
Oh, I see you just changed the `RuntimeError` to a warning; the `tmp` variable is old code.